Unable to build on aarch64

### Description

```
-- The C compiler identification is GNU 15.2.1
-- The CXX compiler identification is GNU 15.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib/distcc/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/lib/distcc/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Python interpreter version 3.13.7
-- Performing Test COMPILER_HAS_MBIG_OBJ
-- Performing Test COMPILER_HAS_MBIG_OBJ - Failed
-- Using (among others) the following compiler flags: -Wall;-Wpedantic;-Wextra;-Wconversion;-Wsign-conversion;-fsigned-char
-- ##################################################
-- Mumble version:              1.5.857
-- Architecture:                arm64
-- Build type:                  None
-- Using LTO in release builds: ON
-- ##################################################
-- gl found | Version: 1.2
-- Looking for sys/cdefs.h
-- Looking for sys/cdefs.h - not found
CMake Error at overlay_gl/CMakeLists.txt:40 (message):
  Can't find the 32bit version of sys/cdefs.h - did you install g++-multilib?

```

There's no support for multilib on aarch64

### Steps to reproduce

Try to build 1.5.857 on a aarch64 system

### Mumble version

1.5.857

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

_No response_

### Relevant log output

```shell

```

### Screenshots

_No response_

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,15 +94,15 @@ option(bundled-utfcpp "Use the bundled utf8cpp version instead of looking for on
 include(compiler)
 include(os)
 
+target_architecture(MUMBLE_TARGET_ARCH)
+string(TOLOWER "${MUMBLE_TARGET_ARCH}" MUMBLE_TARGET_ARCH)
+
 if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
 	option(overlay "Build overlay." ${client})
-	if(64_BIT)
+	if(64_BIT AND NOT MUMBLE_TARGET_ARCH STREQUAL "arm64")
 		option(overlay-xcompile "Build 32 bit overlay library, necessary for the overlay to work with 32 bit processes." ${client})
 	endif()
 endif()
-
-target_architecture(MUMBLE_TARGET_ARCH)
-string(TOLOWER "${MUMBLE_TARGET_ARCH}" MUMBLE_TARGET_ARCH)
 
 message(STATUS "##################################################")
 message(STATUS "Mumble version:              ${PROJECT_VERSION}")

